docs/apache-airflow/authoring-and-scheduling/deferring.rst

```diff
@@ -253,6 +253,73 @@ These parameters can be mapped using the ``expand`` and ``partial`` methods. Not
         trigger_kwargs=[{"moment": timedelta(hours=2)}, {"moment": timedelta(hours=2)}]
     )
 
+Exiting deferred task from Triggers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ .. versionadded:: 2.10.0
+
+If you want to exit your task directly from the triggerer without going into the worker, you can specific operator level attribute ``end_from_trigger`` with the attributes to your deferrable operator other discussed above.
+
+Triggers can have two options: they can either send execution back to the worker or end the task instance directly. If the trigger ends the task instance itself, the ``method_name`` does not matter and can be ``None``. Otherwise, provide the name of the ``method_name`` that should be used when resuming execution in the task.
```

Comment:
```suggestion
Triggers can have two options: they can either send execution back to the worker or end the task instance directly. If the trigger ends the task instance itself, the ``method_name`` does not matter and can be ``None``. Otherwise, provide ``method_name`` that should be used when resuming execution in the task.
```